src/bento/adminData.js

```diff
@@ -29,6 +29,8 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
 export const nodeField = 'numberOfArms'; // Node field
 
 export const nodeLevelAccess = env.NODE_LEVEL_ACCESS; // Node-Level Access is configured as "Off"
```

Comment:
get NODE_LEVEL_ACCESS from siteWideConfig.js. Value is now true/false as opposed to strings

Reply:
That has been resolved in the new PR>